docs/DNS.md

```diff
@@ -10,4 +10,4 @@ The outcome of this is a pods in the cluster can resolve domains like sso.uds.de
 
 ### Nginx Configuration
 
-Additionally, the package includes Nginx configuration that assumes the use of `uds.dev` as the base domain. This configuration is tailored to support the development environment setup, ensuring that Nginx correctly handles requests and routes them within the cluster, based on the `uds.dev` domain.
\ No newline at end of file
+Additionally, the package includes Nginx configuration that assumes the use of `uds.dev` as the base domain. This configuration is tailored to support the development environment setup, ensuring that Nginx correctly handles requests and routes them within the cluster, based on the `uds.dev` domain.
```

Comment:
maybe a silly question but ... what changed in this file?

Reply:
I could have sworn the GitHub diff view made that clearer as I often do the same.  Thanks!